packages/manager/modules/pci/src/projects/project/kubernetes/details/node-pool/scale/scale.routing.js

```diff
@@ -0,0 +1,28 @@
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state(
+    'pci.projects.project.kubernetes.details.nodepools.scale',
+    {
+      url: '/scale?nodePoolId',
```

Comment:
if you need a nodePoolId than why can't you have this as a child route of nodepools.nodepool. By doing this you can avoid nodePoolId in query params.

Reply:
I made it like for the delete component to be consistent.